### PR TITLE
Failed channels stats.

### PR DESCRIPTION
**Pull request by gpolitis**
Exposes the failed channels statistics in colibri stats.

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/Videobridge.java [hunk 1]
```diff
@@ -201,6 +201,13 @@
     private Pattern shutdownSourcePattern;
 
     /**
+     * Keeps track of the total number of failed channels. (look at
+     * @{link RtpChannel} for the definition of "failed").
+     */
+    private Map<MediaType, Integer> totalNumberOfFailedChannelsMap
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Videobridge.java]: Why not just use two atomic integers and remove the synchronization logic?

File: src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java [hunk 2]
```diff
@@ -138,6 +138,18 @@
      */
     public static final String VIDEOSTREAMS = "videostreams";
 
+    /**
+     * The name of the total number of failed audio channels statistic.
+     */
+    private static final String TOTAL_FAILED_AUDIO_CHANNELS
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java]: Can we use names that fit the convention?

**Comment by gpolitis** [comment]
@paweldomas has a better way of doing some of these things here https://github.com/jitsi/jitsi-videobridge/pull/272/commits/b3bdcd81193af19555e78c925475923393fdde16. Will cherry-pick his commit and adapt my changes to his.

File: src/main/java/org/jitsi/videobridge/Conference.java [hunk 3]
```diff
@@ -70,6 +70,12 @@
     private final List<Content> contents = new LinkedList<>();
 
     /**
+     *  A boolean flag that determines whether logging should be enabled for
+     *  this conference.
+     */
+    private final boolean loggingEnabled;
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Conference.java]: This clashes with the implementation in #284 (which implements pretty much the same). Do you prefer to use this version?

**Pull request closed without merging**